.idea/codeStyles/Project.xml

```diff
@@ -1,7 +0,0 @@
-<component name="ProjectCodeStyleConfiguration">
-  <code_scheme name="Project" version="1">
-    <codeStyleSettings language="Dart">
-      <option name="RIGHT_MARGIN" value="80" />
```

Comment:
does it makes sense to keep these maybe? But just change the value to `120`?
That way, if I use some other local setting on my machine, then I still get the project specific settings in this project thanks to these?

Reply:
good idea, I will bring it back